plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -288,13 +296,13 @@ function auto_sizes_filter_render_block_context( array $context, array $block ):
 	return $context;
 }
 
-/**
- * Retrieves the layout settings defined in theme.json.
- *
- * @since n.e.x.t
- *
- * @return array<string, mixed> Associative array of layout settings.
- */
+	/**
+	 * Retrieves the layout settings defined in theme.json.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array<string, mixed> Associative array of layout settings.
+	 */
```

Comment:
```suggestion
/**
 * Retrieves the layout settings defined in theme.json.
 *
 * @since n.e.x.t
 *
 * @return array<string, mixed> Associative array of layout settings.
 */
```